### ISSUE TITLE
Parsing problem when space in source Name

### ISSUE THREAD

**Issue by patti** [JLBLine/PUMA#9]
The output of `make_table.py` puts a space in the 'Name' column values (e.g. 'KGS J...'). If this is fed pack into PUMA, `make_table.py` throws an error because it splits the `input_bayes` file lines on spaces and the indexing gets off. 
- use comma or tab delimiters when writing `output_bayes` file
  or
- add something like `'_'.join(src.names[i].split())` in `calculate_bayes` before writing the file (I tried this but am missing something).

**Comment by JLBLine** [comment]
You are correct this is a right pain in the arse. Will get round to it when I can!
